extension/src/config/init/ensure-resources.js

```diff
@@ -2,8 +2,9 @@ const { ensurePaymentCustomType } = require('./ensure-payment-custom-type')
 const { ensureInterfaceInteractionCustomType } = require('./ensure-interface-interaction-custom-type')
 const { ensureApiExtensions } = require('./ensure-api-extensions')
 const ctpClientBuilder = require('../../ctp/ctp-client')
-const config = require('../../config/config')
+const configLoader = require('../../config/config')
 
+const config = configLoader.load()
 const ctpClient = ctpClientBuilder.get()
 
 function ensureCustomTypes () {
```

Comment:
This should not be disabled too as it is exposed for public usage?

Reply:
This shouldn't be disabled because it is either called from the ensureResources method, or is being specifically invoked via the npm command that runs this method in which case, would you really want to disable it if they passed this variable in? Why would they call this method (using the npm command) to create resources and also want to disable resource creation?

Reply:
I see. Thanks